test/test_logging_config.py

```diff
@@ -44,7 +44,8 @@ def configureLogging(logConfig={}, oneFile=False):
         os.unlink(cfg['logging']['error_log_file'])
     cfg['logging'].update(logConfig)
 
-    girder.logger = girder._setupLogger()
+    girder.logger.setLevel('INFO')
```

Comment:
We generally should never be setting the level of the central `girder.logger`, rather we should prefer to set individual handlers. Why is this being done for this test?

Reply:
Because by default, the level of a logger is 'WARNING' (inherited from the default of the root logger), and our tests expect 'INFO'-level messages to appear in the logs.

Reply:
Interesting point. Given the [behavior of `logging.Logger.setLevel`](https://docs.python.org/3/library/logging.html#logging.Logger.setLevel), I think we should call
`logging.getLogger('girder').setLevel(logging.DEBUG)` immediately after the base Girder logger is created: https://github.com/girder/girder/blob/0c28e5be9f5eaee1f5220930d142908818d2b234/girder/__init__.py#L41

The alternative could be that Girder 'takes over' the Python root logger (`logging.getLogger('girder')`) and sets its level to `logging.NOTSET`. However, particularly in the short term, I think it's good if we discipline ourselves to ensure that everything is logged to `logging.getLogger('girder')` or a descendant logger.